composer.json

```diff
@@ -19,7 +19,8 @@
         "php": ">=5.3.9",
         "doctrine/common": "~2.3",
         "twig/twig": "~1.18",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "symfony/templating": "~2.8"
```

Comment:
We can't add it to the suggest section to keep BC. In 3.0 we can.
The dep in the  framework bundle is not enough, only works if someone install the symfony/framework-bundle package, but if someone uses the full framework requiring symfony/symfony the dep will not be installed.